src/components/tree/tree-item.ts

```diff
@@ -414,17 +413,14 @@ export default class IgcTreeItemComponent extends EventEmitterMixin<
 
   protected render() {
     return html`
-      <div part="wrapper" class=${classMap(this.classes)}>
-        <section part="indentation">
+      <div part="${partNameMap(this.parts)}">
+        <div part="indentation">
           <slot name="indentation"></slot>
-            ${arrayOf(this.level).map(
-              () => html`<span class="tree-node__spacer"></span>`
-            )}
+            ${arrayOf(this.level).map(() => html`<span part="spacer"></span>`)}
```

Comment:
Shouldn't this be inside the slot?

Reply:
Yes, it should be, it must have been the autocomplete. 

PS. @desig9stein the width of the indentation should be set based on the level of the item multiplied by some default width, which should be exposed as a css variable. The iteration should be removed.